cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -785,13 +931,13 @@ if(ARROW_USE_OPENSSL)
   if(NOT TARGET OpenSSL::SSL)
```

Comment:
I think that we can remove this `if` block for CMake < 3.2 because we requires CMake 3.2 or later: https://github.com/apache/arrow/blob/master/cpp/CMakeLists.txt#L18

Reply:
I looked into this and the comment is incorrect. These imported targets were introduced in CMake 3.4.0 https://gitlab.kitware.com/cmake/cmake/blob/75e3a8e811b290cb9921887f2b086377af90880f/Modules/FindOpenSSL.cmake